opentracing/utils.go

```diff
@@ -7,12 +7,13 @@ import (
 )
 
 const (
-	// OpenTracingContextIDHTTPHeaderPrefix precedes the opentracing-related
-	// ContextID HTTP headers.
-	OpenTracingContextIDHTTPHeaderPrefix = "Open-Tracing-Context-Id-"
-	// OpenTracingTagsHTTPHeaderPrefix precedes the opentracing-related
-	// trace-tags HTTP headers.
-	OpenTracingTagsHTTPHeaderPrefix = "Open-Tracing-Trace-Tags-"
+	// ContextIDHTTPHeaderPrefix precedes the opentracing-related ContextID HTTP
+	// headers.
+	ContextIDHTTPHeaderPrefix = "Open-Tracing-Context-Id-"
```

Comment:
FYI, I went with `Open-Tracing` because I was sad when I saw how golang's http libs turned `OpenTracing` into `Opentracing`. Not a good reason, I admit, but there you have it :)

Also, I don't have strong feelings about this -- happy to change it if someone else does!